src/components/common/bottom-link-wrapper/bottom-link-wrapper.styled.ts

```diff
@@ -0,0 +1,7 @@
+import { Typography } from '@mui/material';
+import styled from 'styled-components';
+
+export const BottomLinkWrapper = styled(Typography)`
+  margin-top: 1rem;
+  text-decoration: underline;
```

Comment:
it is better to use a selector [here](https://stackoverflow.com/questions/45841265/react-styled-components-refer-to-other-components)